packages/sanity/src/ui/tab/Tab.tsx

```diff
@@ -0,0 +1,43 @@
+import {ButtonTone, Tab as UITab, TabProps as UITabProps} from '@sanity/ui'
+import React, {forwardRef} from 'react'
+
+/**
+ * @internal
+ *
+ * Icons are not supported in the Studio UI <Tab> component.
+ * Padding and font size are fixed.
+ *
+ */
+export type TabProps = Pick<
+  UITabProps,
+  'id' | 'label' | 'aria-controls' | 'tone' | 'selected' | 'focused'
+> & {
+  size?: 'default' | 'small'
+}
+
+const smallTabProps = {
+  padding: 2,
+  muted: true,
+  tone: 'positive' as ButtonTone,
```

Comment:
Not sure about setting the `tone` here, it won't allow you set a tone different to positive as it will be always overridden.
Maybe we just need to `tone = "positive"` in the default props